### PR TITLE
Add 2020 Jupyter survey to README.md

### PR DESCRIPTION
**Pull request by isabela-pf**
This PR adds
- A list item to the repo's main `readme.md` that links to the 2020 Jupyter Survey data (which is already in this repository)

Thanks in advance for any review! 🌻

### CODE REVIEW

**Review by krassowski — APPROVED** [review]
LGTM

**Comment by isabela-pf** [comment]
Thank you @krassowski! 

I'm flagging for any maintainers that I don't have permissions to merge even approved PRs to this repo (not that I think it's wise to merge my own work anyways) in case there's any confusion. Someone else will have to review and/or merge this if it's wanted.

**Pull request merged**
